### ISSUE TITLE
Can't install through brackets-npm-registry

### ISSUE THREAD

**Issue by entepe85** [zaggino/brackets-eslint#42]
Using Brackets 1.7 (w/ Extract) on OSX 10.11.6 I get the following error upon installation of ESLint through brackets-npm-registry:

`ERROR: "Error: spawn ENOENT\n    at errnoException (child_process.js:980:11)\n    at Process.ChildProcess._handle.onexit (child_process.js:771:34)"`

**Comment by entepe85** [comment]
Both uninstalling and reinstalling the extension and restarting Brackets unfortunately didn't fix the issue.

**Comment by zaggino** [comment]
@entepe85 you can try manual installation, follow the steps here:
https://github.com/zaggino/brackets-npm-registry#how-to-install-brackets-npm-registry
but replace all occurences of `brackets-npm-registry` for `brackets-eslint`
Let me know if that worked for you.

**Comment by zaggino** [comment]
```
cd ~/Library/Application\ Support/Brackets/extensions/user/
git clone https://github.com/zaggino/brackets-eslint brackets-eslint
cd brackets-eslint
npm install
```

**Comment by entepe85** [comment]
Yeah, that fixed it. Seems like stupid me didn't think of the manual installation.

One thing I noticed, though: When I first tried updating brackets-eslint through the npm-registry GUI, the command `typings install` threw an error, so I globally installed the typings package.

**Comment by zaggino** [comment]
That error with typings should be now fixed with version `2.9.5`
